### DISCUSSION TITLE
TuYa TS004F problems

### DISCUSSION THREAD

**Discussion by AVCS21** [Koenkk/zigbee2mqtt#8156] [Devices]
Hello,

I'm trying to include this supported device (https://www.zigbee2mqtt.io/devices/TS004F.html) in zigbee2mqtt add-on in HA, I paired the device correctly but shows me the message "Warning: Device '0x60a423fffe951477' with Zigbee model 'TS004F' and manufacturer name '_TZ3000_xabckq1v' is NOT supported, please follow https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html"

Can anyone help me?

Thanks in advance

**Comment by Koenkk** [comment]
Already supported in zigbee2mqtt 1.20.0

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by AVCS21** [reply]
Thanks, I will try this way.

**Comment by evangelo18** [comment]
I also seem to be having this exact same issue. However I am on latest 1.27 version.

**Reply by Koenkk** [reply]
Can you provide the data/database.db entry of your device?

**Reply by evangelo18** [reply]
{"id":8,"type":"EndDevice","ieeeAddr":"0x2c1165fffec91e17","nwkAddr":2601,"manufId":4098,"manufName":"_TZ3000_czuyt8lz","powerSource":"Battery","modelId":"TS004F","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":260,"inClusterList":[0,1,3,4,6,4096],"outClusterList":[25,10,3,4,5,6,8,4096],"clusters":{"genBasic":{"attributes":{"modelId":"TS004F","manufacturerName":"_TZ3000_czuyt8lz","stackVersion":0,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":64,"stackVersion":0,"hwVersion":1,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1661932294583,"defaultSendRequestWhen":"immediate"}

**Reply by Koenkk** [reply]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by evangelo18** [comment]
thankyou!
